modules/multiplayer/scene_replication_interface.cpp

```diff
@@ -125,9 +125,14 @@ Error SceneReplicationInterface::on_replication_start(Object *p_obj, Variant p_c
 	MultiplayerSynchronizer *sync = Object::cast_to<MultiplayerSynchronizer>(p_config.get_validated_object());
 	ERR_FAIL_COND_V(!sync, ERR_INVALID_PARAMETER);
 
+	const ObjectID oid = node->get_instance_id();
+	MultiplayerSpawner *spawner = rep_state->get_spawner(oid);
+	if (spawner && spawner->get_multiplayer_authority() != sync->get_multiplayer_authority()) {
+		WARN_PRINT("Authority for node \"" + String(sync->get_name()) + "\" does not match its spawner authority and will not sync. Change the authority of a MultiplayerSynchronizer not controlled by the root node instead.");
```

Comment:
```suggestion
	ERR_FAIL_COND_V_MSG(spawner && spawner->get_multiplayer_authority() != sync->get_multiplayer_authority(), ERR_INVALID_PARAMETER, "The authority of the MultiplayerSynchronizer \"" + String(sync->get_path()) + "\" differs from the authority of its \"root_node\" spawner and will not sync. Change the \"root_node\" of the MultiplayerSynchronizer to be a child of the scene root instead.");
```

Not 100% sure about the text, but we should use `ERR_FAIL_COND_V_MSG` instead of the `if` (it's not going to work anyway, we might as well return)

Reply:
Makes sense replacing the warning by an error, I'm commiting your suggestion